src/components/charts/ChartHeaderFilters.tsx

```diff
@@ -43,23 +44,44 @@ function ChartHeaderFilter({
         const app = searchParams.get(QueryParams.AppStoreName)
         const deprecate = searchParams.get(QueryParams.IncludeDeprecated)
         const chartRepoId = searchParams.get(QueryParams.ChartRepoId)
-
+        const registryId = searchParams.get(QueryParams.RegistryId)
+        let isOCIRegistry
         if (key == 'chart-repo') {
             let chartRepoId = selected
-                ?.map((e) => {
-                    return e.value
+                .filter((selectedRepo) => !selectedRepo.isOCIRegistry)
+                ?.map((selectedRepo) => {
+                    return selectedRepo.value
                 })
                 .join(',')
-            let qs = `${QueryParams.ChartRepoId}=${chartRepoId}`
-            if (app) qs = `${qs}&${QueryParams.AppStoreName}=${app}`
-            if (deprecate) qs = `${qs}&${QueryParams.IncludeDeprecated}=${deprecate}`
-            history.push(`${url}?${qs}`)
+
+            let registryId = selected
```

Comment:
Why are we creating the var with same name?

Reply:
Renamed